README.rst

```diff
@@ -442,6 +442,74 @@ In order to switch to this test runner you have set the following in your `setti
 
     TEST_RUNNER = 'openwisp_utils.tests.TimeLoggingTestRunner'
 
+``openwisp_utils.tests.redirect_stdout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This decorator can be used to redirect standard output produced by tests to somewhere else.
+
+Usage example as a decorator:
+
+.. code-block:: python
+
+    import io
+    from openwisp_utils.tests import redirect_stdout
+
+    @redirect_stdout(io.StringIO())
+    def some_test(self):
+        # Pseudo Code
+        print("some output")
+
+    @redirect_stdout(io.StringIO())
+    def some_test(self, stdout):
+        # Pseudo Code
```

Comment:
Instead of pseudo code, I would write something like:

```python
# pseudo code
call_function_which_will_produce_output()
# now you can create assertions on captured output
self.assertIn('expected stdout', stdout.getvalue())
```

Applies to other occurrences as well.